modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -150,11 +150,27 @@ spec:
 <6> The `env` parameter defines a list of Environment Variables that must exist in all containers in the pod created by OLM.
 <7> The `envFrom` parameter defines a list of sources to populate Environment Variables in the container.
 <8> The `volumes` parameter defines a list of Volumes that must exist on the pod created by OLM.
-<9> The `volumeMounts` parameter defines a list of VolumeMounts that must exist in all containers in the pod created by OLM. If a `volumeMount` references a `volume` that does not exist, OLM fails to deploy the Operator.
+<9> The `volumeMounts` parameter defines a list of volume mounts that must exist in all containers in the pod created by OLM. If a `volumeMount` references a `volume` that does not exist, OLM fails to deploy the Operator.
 <10> The `tolerations` parameter defines a list of Tolerations for the pod created by OLM.
 <11> The `resources` parameter defines resource constraints for all the containers in the pod created by OLM.
 <12> The `nodeSelector` parameter defines a `NodeSelector` for the pod created by OLM.
 
+. If the cluster is in STS mode, include the following fields in the `Subscription` object:
```

Comment:
I have a general comment on "STS mode", personally I have a little concern user may be confused by "STS mode" and "manual mode".And I searched "STS mode" in OCP4.13 document but found no result. 
+ @jeana-redhat for comment

Reply:
Yes, I also worry about the inconsistency, and that eventually this will not be a feature for only AWS, so the terminology should be generalized. Right now, the GUI says "STS Mode", so I think for GUI steps it makes sense to match the text the user will see.

Last week, I proposed using platform-neutral language that is consistent with existing CCO terminology in the GUI, and @gallettilance was onboard with it. But I don't know if that is a change that can happen before 4.14 or not.

Reply:
Could change the wording here in the meantime to be more specific like:
```suggestion
. If the web console shows that the cluster is in "STS mode", include the following fields in the `Subscription` object:
```

Reply:
@adellape That change makes sense to me until the language can be updated post-4.14 in the console.